commands/api.go

```diff
@@ -193,8 +196,8 @@ func apiCommand(cmd *Command, args *Args) {
 	isGraphQL := path == "graphql"
 	if isGraphQL && params["query"] != nil {
 		query := params["query"].(string)
-		query = strings.Replace(query, quote("{owner}"), quote(owner), 1)
```

Comment:
What is the purpose of using `quote()` here? Isn't replacing a word with quotes with another word with quotes the same as just replacing a word with another word?

I am not familiar with GraphQL syntax so I might be missing something here :)

Reply:
It looks like these 2 lines are the only places invoking `quote()`. So if this conversation is resolved with the decision to not use it, I can remove the function definition at the bottom of the file.

Reply:
The reason I chose to replace only the quoted `"{owner}"` and `"{repo}"` placeholders  was to indicate that these end up being hardcoded string values within the GraphQL query, rather than accidentally replacing the non-quoted placeholder `{owner}` somewhere where it's not its own string literal. Additionally, if the expanded values contained a literal quote `"` themselves, this would escape it. (It's impossible for either owner handle or repo name to contain special characters such as a quote, though.)

I'm not sure whether this guard or limitation actually pays off, and it definitely prevents a person from using both `{owner}` and `{repo}` within the same string literal. Maybe you're right and we should go the simpler way and drop the `quote()` for now.